packages/@aws-cdk-testing/cli-integ/resources/cdk-apps/app/app.js

```diff
@@ -830,9 +841,7 @@ switch (stackSet) {
     new DockerInUseStack(app, `${stackPrefix}-docker-in-use`);
     new DockerStackWithCustomFile(app, `${stackPrefix}-docker-with-custom-file`);
 
-    new NotificationArnPropStack(app, `${stackPrefix}-notification-arn-prop`, {
-      notificationArns: [`arn:aws:sns:${defaultEnv.region}:${defaultEnv.account}:${stackPrefix}-test-topic-prop`],
```

Comment:
Moved this to the stack level, which now reads it from an env variable to avoid hard coding stuff.